keyboards/dztech/dz65rgb/keymaps/adi_dz65rgb_ansi/readme.md

```diff
@@ -0,0 +1,13 @@
+# Adi's DZ65RGB Layout
+[Layout Visual](https://i.imgur.com/xuFkvVI.png)
+
+This layout modifies the extra row on the right and makes some of the key bindings easier to reach on Layer 1.
+## Step by Step
+1. Build your Firmware: run build command in directory
+```make dztech/dzrgb:adi_dz65rgb_ansi```
+2. Flash your Firmware:
+    * Open QMK Toolbox Application
+    * Open firmware file: adi_dz65rgb_ansi.bin
```

Comment:
The compiled firmware includes the name of the keyboard for which it was compiled:

![image](https://user-images.githubusercontent.com/18669334/72134798-be796280-3339-11ea-9325-26daf1a0a9d7.png)

```suggestion
    * Open firmware file: dztech_dz65rgb_v1_adi_dz65rgb_ansi.bin
```